include/swift/Runtime/Win32Strings.h

```diff
@@ -0,0 +1,35 @@
+//===--- Win32Strings.h ---------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Includes conversion functions for use with Windows API that produces or
+// consumes wide strings (wchar_t) which, on Windows, are UTF-16.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_RUNTIME_WIN32STRINGS_H
+#define SWIFT_RUNTIME_WIN32STRINGS_H
+
+#if defined(_WIN32)
+namespace swift {
+/// Convert an argument, represented by a wide string, to UTF-8.
+///
+/// @param str The string to convert.
+///
+/// @returns The string, converted to UTF-8. The caller is responsible for
+///   freeing this string when done with it.
```

Comment:
Can you specify how? "free()ing" would suffice, assuming that's the right call for it.

Reply:
Will update the comment.